lib/libfrr.c

```diff
@@ -36,8 +36,8 @@ const char frr_sysconfdir[] = SYSCONFDIR;
 const char frr_vtydir[] = DAEMON_VTY_DIR;
 const char frr_moduledir[] = MODULE_PATH;
 
-char frr_protoname[] = "NONE";
-char frr_protonameinst[] = "NONE";
+char frr_protoname[] = "NONE ";
+char frr_protonameinst[] = "NONE ";
 
```

Comment:
oops, my bad.

but the fix is not right either ;)
need to make the arrays [256] or so.

(it'll also truncate the instance number in ospf...)